.gitignore

```diff
@@ -21,6 +21,8 @@ xcuserdata/
 *.moved-aside
 *.xccheckout
 *.xcscmblueprint
+.DS_Store
+**/.DS_Store
```

Comment:
@BIG-RAT I _think_ it may be enough to add here
`*.DS_Store`
instead of those two entries.

Reply:
No need for the * in the front at all.  Simply `.DS_Store` will ignore all .DS_Store files in all folders.